### DISCUSSION TITLE
Make the sticky send after x amount of sec.

### DISCUSSION THREAD

**Discussion by godgutten** [andrerahardjo97/discord-sticky-message-bot#34] [Ideas]
Hey. I would love to have it so the bot always stays at the bottom. Atm, you need to send X amount of messages before it gets resent. But that's not really handy. It would be much better if it gets resent 20 to 30 sec after the last message has ben sent. So say people keep typing then it sends also after an X amount of messages. This will make sure that it will always be at the bottom but also make sure that we don't spam the API but also makes sure all the chaters in the chat also read it.

Would love it if this is something you can implement.

**Comment by andrerahardjo97** [comment]
Great idea!
I would love to implement your idea, but right now I just join a new company. 😁
I probably going to implement this after my probation period.
